js/apps/account-ui/src/utils/useFormatDate.ts

```diff
@@ -0,0 +1,16 @@
+const DATE_AND_TIME_FORMAT: Intl.DateTimeFormatOptions = {
+  dateStyle: "long",
+  timeStyle: "short",
+};
+
+//todo use user local
+export default function useFormatDate() {
+  return {
+    formatDate: function (
+      date: Date,
+      options: Intl.DateTimeFormatOptions | undefined = DATE_AND_TIME_FORMAT,
+    ) {
+      return date.toLocaleString("en", options);
+    },
+  };
+}
```

Comment:
Since there is only one function being returned in this object, let's reduce the complexity here by returning the function directly.

```suggestion
export default function useFormatDate() {
  return function formatDate(
    date: Date,
    options: Intl.DateTimeFormatOptions | undefined = DATE_AND_TIME_FORMAT,
  ) {
    return date.toLocaleString("en", options);
  };
}
```